migration/iD10/oracle/202203151430_IDEMPIERE-5232.sql

```diff
@@ -106,35 +106,10 @@ UPDATE AD_Field SET SeqNo=390, AD_Reference_Value_ID=NULL, AD_Val_Rule_ID=NULL,
 UPDATE AD_ViewColumn SET ColumnSQL='COALESCE(f.ad_chart_id, c.ad_chart_id)',Updated=TO_DATE('2022-03-15 14:33:06','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=0 WHERE AD_ViewColumn_ID=200277
 ;
 
```

Comment:
why is this changing an old migration script (from march 22)?  That would potentially break the implementations following master that already applied it

Reply:
the view is recreated in the 202209141520_IDEMPIERE-5396.sql script